Gear.go

```diff
@@ -0,0 +1,101 @@
+package connect
+
+import (
+	"fmt"
+)
+
+// Gear describes a Garmin Connect gear entry
+type Gear struct {
+	Uuid            string  `json:"uuid"`
+	GearPk          int     `json:"gearPk"`
+	UserProfileID   int64   `json:"userProfilePk"`
+	GearMakeName    string  `json:"gearMakeName"`
+	GearModelName   string  `json:"gearModelName"`
+	GearTypeName    string  `json:"gearTypeName"`
+	DisplayName     string  `json:"displayName"`
+	CustomMakeModel string  `json:"customMakeModel"`
+	ImageNameLarge  *string `json:"imageNameLarge"`
```

Comment:
Do you have strong opinions about `*string`? The rest of the library uses `string` with the zero value used for "missing strings". I would suggest that the `Gear` type does the same - or we change it all other places too.

This goes for other pointer-members too.

Reply:
I can't really recall why I did this, I should have kept more granular commit history.